src/defaults.json

```
@@ -627,11 +627,7 @@
 			}
 		]
 	},
-	"PasswordInput": {
-		"label": "Tell me your secrets",
-		"errorMessage": "Please enter a valid name"
-	},
-	"PasswordInputStateful": {
+	"PasswordInpu": {
```

```suggestion
	"PasswordInput": {
```
Mr. T was not invited ☹️

---

😄